packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -1,23 +1,36 @@
 'use client';
+
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { styled, useTheme, SxProps } from '@mui/material';
+
+// MUI: Styles & Theme
+import { styled, useTheme, SxProps } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import type {} from '@mui/material/themeCssVarsAugmentation';
+
+// MUI: Components
```

Comment:
No need to add all these comments, for consistency with other files in the project.